src/ConEmu/GuiServer.cpp

```diff
@@ -474,7 +474,7 @@ BOOL CGuiServer::GuiServerCommand(LPVOID pInst, CESERVER_REQ* pIn, CESERVER_REQ*
 					};
 				} arg = {hConWnd, pIn->hdr.nSrcPID, pIn->SrvStartStop.dwKeybLayout, nStartTick};
 
-				gpConEmu->CallMainThread(true, arg.OnSrvStarted, (LPARAM)&arg);
+				gpConEmu->CallMainThread(true, arg::OnSrvStarted, (LPARAM)&arg);
```

Comment:
The `arg` is not a type, it's an instance. So `arg.OnSrvStarted` is OK, but `arg::OnSrvStarted` would not be compiled ever.